frontend/packages/ux-editor/src/components/Properties/EditSubformTableColumns/ColumnElement/ColumnElement.tsx

```diff
@@ -12,19 +12,21 @@ export type ColumnElementProps = {
   layoutSetName: string;
   tableColumn: TableColumn;
   columnNumber: number;
+  initialOpenForEdit: boolean;
```

Comment:
What about changing the name to `isColumnInitiallyEditable` or `isInitialOpenForEdit`

Reply:
Thx! I will use one of your suggestions :)